### ISSUE TITLE
PyQS + Python 3.9 broken?

### ISSUE THREAD

**Issue by alexef** [spulec/PyQS#74]
I'm trying to run PyQS, but failing with:

```
python3 -m venv .venv
./.venv/bin/pip install pyqs
./.venv/bin/pyqs test
Traceback (most recent call last):
  File "/Users/alex/./.venv/bin/pyqs", line 33, in <module>
    sys.exit(load_entry_point('pyqs==0.1.5', 'console_scripts', 'pyqs')())
  File "/Users/alex/.venv/lib/python3.9/site-packages/pyqs/main.py", line 112, in main
    _main(
  File "/Users/alex/.venv/lib/python3.9/site-packages/pyqs/main.py", line 145, in _main
    manager.start()
  File "/Users/alex/.venv/lib/python3.9/site-packages/pyqs/worker.py", line 354, in start
    child.start()
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 284, in _Popen
    return Popen(process_obj)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_spawn_posix.py", line 32, in __init__
    super().__init__(process_obj)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/popen_spawn_posix.py", line 47, in _launch
    reduction.dump(process_obj, fp)
  File "/usr/local/Cellar/python@3.9/3.9.1_6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)
_pickle.PicklingError: Can't pickle <class 'botocore.client.SQS'>: attribute lookup SQS on botocore.client failed
```
boto:
```./.venv/bin/pip freeze                         
boto3==1.16.56
botocore==1.19.56```

**Comment by spulec** [comment]
Thanks for opening. I don't think this is related to Py3.9 as I was able to recreate on 3.8. It looks like we are pickling the boto3 connections which isn't going to work.

I've opened a PR with a refactor of the connection creation here which should solve the issue: https://github.com/spulec/PyQS/pull/75

**Comment by spulec** [comment]
I've merged #75 and released a new version as `0.1.6`. Let me know if there are any issues.
